app/konnect/configure/runtime-manager/runtime-instances/troubleshoot.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Troubleshoot Runtime Instances
+no_version: true
+---
+
+## Out of sync runtime instance
+
+Occasionally, a {{site.base_gateway}} runtime instance might get out of sync
+with the {{site.konnect_short_name}} control plane. If this happens, you would
+see the status `Out of sync` on the Runtime Instances page, meaning the control
+plane can't communicate with the instance.
+
+Troubleshoot the issue using the following steps:
+
+1. Ensure the runtime instance is running. If it's not running, start it; if it
+is running, restart it.
+
+    Check the sync status in the Runtime Manager.
+
+1. Check the logs of the runtime that's appearing as `Out of sync`.
+{{site.base_gateway}} logs can be found at `<kong-prefix-directory>/logs`,
+where the default directory is `/usr/local/kong/logs`.
```

Comment:
Would it make more sense to say: 
"The default directory for Kong Gateway logs is `/usr/local/kong/logs`?